vncviewer/Viewport.cxx

```diff
@@ -598,13 +600,40 @@ int Viewport::handle(int event)
   case FL_DRAG:
   case FL_MOVE:
   case FL_MOUSEWHEEL:
-    buttonMask = 0;
+
+    // FIXME: FLTK is not consistent in how it sends FL_PUSH/FL_RELEASE
+    // events:
```

Comment:
You could consider making the workaround separate commits, to make things more clear. Assuming it works okayish without the workarounds in place. We don't want the commit without a workaround to be completely broken.